src/test/kotlin/com/zepben/evolve/services/network/testdata/TestDataCreators.kt

```diff
@@ -195,6 +196,19 @@ fun createFeeder(networkService: NetworkService, mRID: String, name: String, sub
             }
         }
 
+fun createLvFeeder(
+    networkService: NetworkService,
+    mRID: String,
+    name: String,
+    headTerminal: Terminal? = null
+): LvFeeder =
+    LvFeeder(mRID).apply {
+        this.name = name
+        normalHeadTerminal = headTerminal
+
+        networkService.add(this)
+    }
+
```

Comment:
question: These function are mmainly left over constructs from when this was all Java code and being used in large, clunky tests. Do you feel with your new tests that this function adds value? 

I suspect `createLvFeeder(ns, "id", "name", headTerminal)` can be replaced with `LvFeeder("id"){ normalHeadTerminal = headTerminal }.also( ns.add(it) }`, as I suspect all new tests are not relying on the name.